Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -59,10 +59,10 @@ def parser(cls, parser, loggers=None):
         )
 
         parser.add_argument(
-            '--revert-issue',
-            dest='revert_issue',
+            '--reason', '-re',
```

Comment:
Why not just `-r`?

Reply:
Ah! Then we're better off just dropping this. We don't have precedence for a 2 character short argument like this. Another thing you could do (see `git-webkit clone`) is have `reason` be the primary argument, but also respect `--why` and `-w`. That doesn't change your existing code, just the possible arguments a user could provide on the command line.